web/src/tippyjs.js

```diff
@@ -567,4 +567,14 @@ export function initialize() {
         },
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: "#send_later",
+        delay: [500, 0],
+        content: $t({
+            defaultMessage: "Send later",
+        }),
```

Comment:
Do we need to move the content here, or could it stay in the template? Mostly just thinking about what is more readable.

Reply:
Okay, moved to the template.